### ISSUE TITLE
Using retrolambda in project library causes databinding error

### ISSUE THREAD

**Issue by alekmiel** [evant/gradle-retrolambda#118]
When using retrolambda and databinding together in a library it gives an error. However without retrolambda in project library everything is working.

Link to repository with example: https://github.com/alekmiel/test-multimodule-binding

Error:

```
D:\Projekty\TestMultimoduleBinding\app\src\main\java\testmultimodulebinding\MainActivity.java:8: error: package testmultimodulebinding.databinding does not exist
import testmultimodulebinding.databinding.ActivityMainBinding;
                                         ^
warning: Supported source version 'RELEASE_7' from annotation processor 'android.databinding.annotationprocessor.ProcessDataBinding' less than -source '1.8'

 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileDebugJavaWithJavac'.
> java.lang.RuntimeException: Found data binding errors.
****/ data binding error ****msg:Cannot find the setter for attribute 'app:integerSupplier' with parameter type testmultimodulebinding.IntegerSupplier. file:D:\Projekty\TestMultimoduleBinding\app\src\main\res\layout\activity_main.xml loc:20:35 - 20:59 ****\ data binding error ****
```

**Comment by runforprogram** [comment]
I also meet this issue,and i find not any useful info with google. by the error log message,it like mean that the databinding only support Java 1.7,but the  `retrolambda` must use Java 1.8?

**Comment by evant** [comment]
@runforprogram that warning can be safely ignored, since java 8 should support java 7 processors fine. The real issue here is that the `@BindingAdapter ` defined in the library project is not being picked up by the app project.

**Comment by evant** [comment]
This is due to databinding using non-class files in it's build output. Previous versions of retrolamba did not copy these files over when doing processing, but it was fixed in `2.0.4`. (See https://github.com/orfjackal/retrolambda/issues/54). You can simply point to a newer version of retrolamba by doing
```groovy
dependencies {
    retrolambdaConfig 'net.orfjackal.retrolambda:retrolambda:2.0.4'
}
```
or wait until the next release which will default to the latest version.
